File: Udemy/Algoritmos e Logica de Programacao/C++/Estrutura Sequencial/ProblemaCirculo.cpp
/*
Problema "circulo"
Fazer um programa para ler o valor "r" do raio de um círculo, e depois mostrar o valor da área do
círculo com três casas decimais. A fórmula da área do círculo é a seguinte: 𝑎𝑟𝑒𝑎 = 𝜋. 𝑟􀬶. Você pode
usar o valor de 𝜋 fornecido pela biblioteca da sua linguagem de programação, ou então, se preferir, use
diretamente o valor 3.14159.
*/


#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main(){
	
	double area, raio;
	
	
	cout << "Digite o valor do raio do circulo: ";
	cin >> raio;
	area = M_PI * pow(raio,2);
	cout << fixed << setprecision(3);
	cout << "AREA = " << area;


    return 0;
}

File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaSequencial/src/application/ProblemaCirculo.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaCirculo {

	public static void main(String[] args) {
		/*
		 * Problema "circulo"
Fazer um programa para ler o valor "r" do raio de um círculo, e depois mostrar o valor da área do
círculo com três casas decimais. A fórmula da área do círculo é a seguinte: Você pode
usar o valor de fornecido pela biblioteca da sua linguagem de programação, ou então, se preferir, use
diretamente o valor 3.14159.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double raio, area;
		
		System.out.print("Digite o valor do raio do circulo: ");
		raio = sc.nextDouble();
		area = Math.PI * Math.pow(raio,2);
		System.out.println("AREA = "+String.format("%.3f", area));
		
		sc.close();

	}

}
